tests/kubesonic/kubesonic_test.md

```diff
@@ -0,0 +1,105 @@
+# MCLAG Test Plan
+
+## Rev 0.1
+
+- [Revision](#revision)
+- [Overview](#overview)
+  - [Scope](#scope)
+    - [Testbed](#testbed)
+    - [k8s version](#k8s-version)
+    - [Supported hwsku](#supported-hwsku)
+- [Topology](#topology)
+- [Test DUT join and disjoin from k8s cluster](#test-dut-join-and-disjoin-from-k8s-cluster)
+    - [Setup](#setup)
+      - [Install Minikube and start Minikube](#install-minikube-and-start-minikube)
+      - [Update the server kernel parameter](#update-the-server-kernel-parameter)
+      - [Update the kubelet configmap](#update-the-kubelet-configmap)
+      - [Deploy the daemonset](#deploy-the-daemonset)
+      - [Prepare the cert](#prepare-the-cert)
+      - [Prepare Minikube VIP DNS](#prepare-minikube-vip-dns)
+    - [Trigger DUT to join](#trigger-dut-to-join)
+    - [Trigger DUT to be deployed daemonset pod](#trigger-dut-to-be-deployed-daemonset-pod)
+    - [Trigger daemonset pod to be removed from the DUT](#trigger-daemonset-pod-to-be-removed-from-the-dut)
+    - [Trigger DUT to disjoin](#trigger-dut-to-disjoin)
+    - [Teardown](#teardown)
+      - [Clean up](#clean-up)
+      - [Keep the Minikube cluster running](#keep-the-minikube-cluster-running)
+    - [Server lockup](#server-lockup)
+
+
+## Revision
+
+| Rev |     Date    |       Author          |         Change Description         |
+|:---:|:-----------:|:---------------------:|:----------------------------------:|
+| 0.1 |  12/25/2024 |         Yun           |          Initial version           |
+
+## Overview
+
+The purpose is to test the functionality of KubeSonic feature on the SONIC switch DUT. The tests expecting that SONIC can join a k8s cluster and can be deployed a k8s daemonset pod.
```

Comment:
"can be deployed a k8s daemonset pod" ?  is that "can be deployed a k8s worker" ?